src/server/api.js

```diff
@@ -72,4 +72,25 @@ function submitApplication(req, res) {
   }
 }
 
-module.exports = { submitApplication };
+
+function getApplicationStatus(req, res) {
+  const id = req.params.id;
+  if (id === undefined) {
+    // TODO(all) what should be returned here?
+    res.status(500).json({ error: 'need id' });
+  }
+  const request = {
+    formSubmissionId: id
+  };
+  voaService.getFormSubmissionStatus(request, (err, response) => {
+    if (err) {
+      debug(`voaService response had error ${err}`);
+      // TODO(awong): This may leak server config info on error. Is that a problem?
+      res.status(500).send({ error: err });
```

Comment:
pretty sure that .json is a syntactic sugar for forcing something non JSON to json, send accepts all formats and pivots the headers based on the response.
This is my general understanding